core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -24,6 +24,31 @@ var ExplorationEditorTranslationTab = function() {
   var translationWelcomeModal = element(
     by.css('.protractor-test-translation-tab-welcome-modal'));
 
+  this.saveUploadedAudioButton = element(
+    by.css('.protractor-test-save-button'));
+
+  this.wrongFileTypeErrorMessage = function() {
+    return (element(by.css('div.error-message')).getText());
+  };
+
+  this.getColorOfInitNode = function() {
```

Comment:
Can we make this on the basis of stateName? (not sure)

Reply:
@DubeySandeep this is sadly not possible :(

Reply:
Can you please check how these methods work:
https://github.com/oppia/oppia/blob/574932964e4393f5918f09f7678e9a39d5d07cdc/core/tests/protractor_utils/ExplorationEditorMainTab.js#L930-L942

Reply:
done.
Thanks for the reference. :)